CONTRIBUTING.md

```diff
@@ -127,6 +127,38 @@ so as not to disturb any real `addons.mozilla.org` data.
   then you'd have to use our staging API server. File an issue for information
   on that.
 
+## Working on CLI
+
+WebExtensions rely on [Yargs](http://yargs.js.org) to parse its commands and
+their options. The commands are defined in `src/program.js`, `main` function
+describes them and `program.execute` function defines what happens with parsed
+commands. `cmd` directory is where all command actions are stored, options are
+passed to the corresponding functions as arguments.
+
+### Adding a command option
+
+To add a command option locate the relevant command description (i.e. `run`).
+Here is an example (note the trailing comma):
+
+````
+'new-option': {
+  describe: 'A wonderful new option with useful functionality.',
+  alias: ['alias'],
+  demand: false,
+  requiresArg: true,
+  type: 'boolean',
+  coerce: nameOfCoerceFunction,
+},
```

Comment:
This looks good but it's a little bit abstract which will make it hard for someone to understand. If it were me, I'd be wondering what is `new-option` and what is `alias`? I would suggest trying to make it a bit more realistic, like:
````
'file-path': {
  describe: 'An absolute file path.',
  alias: ['fp'],
  demand: false,
  requiresArg: true,
  coerce: nameOfCoerceFunction,
},
````
That way you can refer to it as `--file-path=./path/to/file` which looks more like a realistic option. You can talk about `--fp` as the alias which makes a bit more sense. I also removed the `boolean` type because `requiresArg: true` contradicts that.